keyboards/keychron/q8/readme.md

```diff
@@ -0,0 +1,19 @@
+# Keychron Q8
+
+A customizable 65% Alice layout keyboard.
+
+* Keyboard Maintainer: [Keychron](https://github.com/keychron)
+* Hardware Supported: Keychron Q8
+* Hardware Availability: [Keychron](https://www.keychron.com)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make keychron/q8/q8_ansi_stm32l432:default
```

Comment:
Is this name correct ? I don't see any folder `keychron/q8/q8_ansi_stm32l432` but only `keychron/q8/q8_ansi`?

Reply:
Should just be `ansi`.

Reply:
Thanks, corrected now.